package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@signalk/nmea0183-utilities",
-  "version": "0.6.0",
+  "version": "0.7.0",
   "description": "Various utilities for transforming NMEA0183 units into SI units for use in SK.",
```

Comment:
I’d like to keep version bumps separate commits. You don’t know when the commit will be part of a published package. If you want to indicate how a commit or pr should be treated per semantic versioning one way to do that would be https://github.com/conventional-changelog/standard-version : write strictly formatted messages.